frontend/src/templates/Field/Table/TableField.tsx

```diff
@@ -139,6 +142,13 @@ export const TableField = ({
           </Tbody>
         </Table>
       </Box>
+      {isMobile ? null : (
+        // Error will only render if table field has errors. Since the only possible errors are
+        // for required fields (for now), just render a hardcoded error message.
+        <FormErrorMessage my="0.75rem">
+          Please fill in the required fields.
```

Comment:
what might be a more sustainable solution here? perhaps `unique` the error messages and display all?

Reply:
I fear that would be too confusing. I will unique and show the first error I guess. Will eventually allow users to fix all the errors.

Reply:
[2da28ab](https://github.com/opengovsg/FormSG/pull/3577/commits/2da28ab18d00018a43e933e7138daf792b52a956)